napari/layers/labels/labels.py

```diff
@@ -1142,9 +1151,13 @@ def get_color(self, label):
         elif label is None or (
             self.show_selected_label and label != self.selected_label
         ):
-            col = self.colormap.map([0, 0, 0, 0])[0]
+            col = self.colormap.map([self._background_label])[0]
```

Comment:
This also should work, but maybe less obvious.
```suggestion
            col = self.colormap.map(self._background_label)[0]
```